src/logout.js

```diff
@@ -33,7 +33,11 @@ export async function logout({ method, redirect } = {}) {
 }
 
 async function completeSamlLogout() {
-  if (!store.tokens.accessToken) return;
+  if (!store.tokens.accessToken) {
+    throw new Error(
+      "Access token is required to complete SAML logout process. Please log in to obtain token."
+    );
```

Comment:
Who is this error intended for? The end user? This is way beyond what most people will understand.

Something like "Please log in to authorize your logout request" would be much less confusing.